arm-image-installer

```diff
@@ -510,6 +499,59 @@ get_lv_name () {
 	LV_NAME=$(lvs --devicesfile "" --devices "$MEDIA" --devices "$ROOTPART" -o lv_name --noheadings "$LVM_NAME" 2>/dev/null | tr -d ' ')
 }
 
+detect_image_variant () {
```

Comment:
I am overall fine with this but I can't help we could simplify the logic overall.